src/App.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import './App.css';
+import Tab from './components/Tab';
+
+class App extends React.Component {
```

Comment:
Also, do not directly write into the app component.
What happens if you have to add more examples ?